### DISCUSSION TITLE
Add a black backround that measures 8300x7600 pixels,

### DISCUSSION THREAD

**Discussion by Talo07** [ImageMagick/ImageMagick#4336] [Help]
Dear colleagues,

I am new to ImageMagick. I need your help with my * .tif format image files

I would like to add a black color background which measures 8300x7600 pixels (without changing the colors of my original images). Output must always be a * .tif  format containing my original image with the background to create.

I would be grateful,
Thanks in advance,
Aouf

**Comment by fmw42** [comment]
What is your ImageMagick version and platform/OS. Syntax may differ.  Please post and example tif, since tif may contain more than one page, so that may change the command.
